src/main/menu/get-app-menu-items.injectable.ts

```diff
@@ -0,0 +1,333 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { checkForUpdates } from "../app-updater";
+import { docsUrl, productName, supportUrl } from "../../common/vars";
+import { exitApp } from "../exit-app";
+import { broadcastMessage } from "../../common/ipc";
+import { openBrowser } from "../../common/utils";
+import { showAbout } from "./menu";
+import windowManagerInjectable from "../window-manager.injectable";
+import {
+  BrowserWindow,
+  MenuItem,
+  MenuItemConstructorOptions,
+  webContents,
+} from "electron";
+import loggerInjectable from "../../common/logger.injectable";
+import appNameInjectable from "../app-paths/app-name/app-name.injectable";
+import electronMenuItemsInjectable from "./electron-menu-items.injectable";
+import isAutoUpdateEnabledInjectable from "../is-auto-update-enabled.injectable";
+import navigateToPreferencesInjectable from "../../common/front-end-routing/routes/preferences/navigate-to-preferences.injectable";
+import navigateToExtensionsInjectable from "../../common/front-end-routing/routes/extensions/navigate-to-extensions.injectable";
+import navigateToCatalogInjectable from "../../common/front-end-routing/routes/catalog/navigate-to-catalog.injectable";
+import navigateToWelcomeInjectable from "../../common/front-end-routing/routes/welcome/navigate-to-welcome.injectable";
+import navigateToAddClusterInjectable from "../../common/front-end-routing/routes/add-cluster/navigate-to-add-cluster.injectable";
+import isMacInjectable from "../../common/vars/is-mac.injectable";
+
+function ignoreIf(check: boolean, menuItems: MenuItemConstructorOptions[]) {
+  return check ? [] : menuItems;
+}
+
+export interface MenuItemsOpts extends MenuItemConstructorOptions {
```

Comment:
nit: the name probably should be `MenuItemOpts`

Reply:
True. Changed.